### ISSUE TITLE
Implement alternatives to long press on desktop

### ISSUE THREAD

**Issue by gspencergoog** [flutter/flutter#36314]
For `ListTile`, and `ReorderableListView`, there are handlers that support long press on mobile platforms.  These don't work well on desktop, and so they need to instead support other affordances.

These affordances will need to be designed and implemented to fit with the Material Design spec.

**Comment by ZhennanWu** [comment]
No, it's not just for desktop convenience. Because the current long press implementation isn't even what recommended by material design. [WHAT MATERIAL DESIGN RECOMMENDED](https://material.io/components/lists/#behavior) is a icon handler dedicated to reordering. I'm not saying long press reorder is evil, but it bothers me why flutter:material's ReorderableListView doesn't have this out of the box.
